README.md

```diff
@@ -165,22 +164,23 @@
 - [Brackets](https://github.com/adobe/brackets) - Modern, open source text editor that understands web design. 👏
 - [neovim](https://github.com/neovim/neovim) - Vim-fork focused on extensibility and usability. 👏
 - [Oni](https://github.com/onivim/oni) - Modern Modal Editing - powered by Neovim. 👏
+- [Oni2](https://github.com/onivim/oni2) - Native, lightweight modal code editor. 👏
 - [Sublime Text](https://www.sublimetext.com/) - Sophisticated text editor for code, markup and prose.
 - [Vim](https://github.com/vim/vim) - The ubiquitous text editor. 👏
-- [Visual Studio Code](https://github.com/Microsoft/vscode) - Free, open source and runs everywhere. 👏
+- [Visual Studio Code](https://github.com/Microsoft/vscode) - Free, open source, powerful and runs everywhere. 👏
 
 ## Version Control
 
 - [Aurees](https://aurees.com/) - Easy, fast and free.
 - [CodeReview](https://github.com/FabriceSalvaire/CodeReview) - Git GUI tool to perform code review. 👏
 - [GitAhead](https://github.com/gitahead/gitahead) - Graphical Git client designed to help you understand and manage your source code history. 👏
-- [Git Cola](https://github.com/git-cola/git-cola) - The highly caffeinated Git GUI. 👏
+- [Git Cola](https://github.com/git-cola/git-cola) - Powerful Git GUI with a slick and intuitive user interface. 👏
 - [Git Extensions](https://github.com/gitextensions/gitextensions) - Standalone UI tool for managing git repositories. 👏
 - [Git Horizon](https://www.cycligent.com/git-tool) - Cross-platform, Actionable, Graphical Git Dashboard.
-- [GitKraken](https://www.gitkraken.com/) - The most popular Git GUI for Linux.
+- [GitKraken](https://www.gitkraken.com/) - The legendary Git GUI client for Linux.
```

Comment:
Words like `legendary` are useless. It doesn't describe what its strengths are. Also don't use `Linux` in the descriptions.